test/utils/describeConformanceV5.js

```diff
@@ -251,14 +280,42 @@ const fullSuite = {
 export default function describeConformanceV5(minimalElement, getOptions) {
   const { after: runAfterHook = () => {}, only = Object.keys(fullSuite), skip = [] } = getOptions();
 
+  const filteredTests = Object.keys(fullSuite).filter(
+    (testKey) => only.indexOf(testKey) !== -1 && skip.indexOf(testKey) === -1,
+  );
+
+  describe('Test Validator', () => {
+    it('should throw error if run the tests without required property', () => {
+      expect(() => validateOptions({}, ['themeVariants'])).to.throw();
+      expect(() => validateOptions({}, ['themeStyleOverrides'])).to.throw();
+      expect(() => validateOptions({}, ['themeDefaultProps'])).to.throw();
+    });
+
+    it('should not throw error if provided required properties (does not care the type)', () => {
```

Comment:
How about we keep a tight correlation between the required props for the tests and the validation of them? This way, we can remove code for specific tests at the same time

Reply:
Yep, I am good with that.